test/cnudie/cnudie_util_test.py

```diff
@@ -33,7 +39,56 @@ def cid():
 
 
 def comp(name, version) -> cm.Component:
-    return cm.Component(name, version, [],[],[],[],[],[])
+    return cm.Component(
+        name=name,
+        version=version,
+        provider=cm.Provider.INTERNAL,
+        repositoryContexts=[],
+        componentReferences=[],
+        sources=[],
+        resources=[],
+        labels=[],
+    )
+
+
+def comp_desc(name, version) -> cm.ComponentDescriptor:
+    return cm.ComponentDescriptor(
+        meta=cm.Metadata(),
+        component=comp(name, version),
+    )
+
+
+def create_ctf(
+    output_filename: str,
+    component_descriptors: typing.List[cm.ComponentDescriptor],
+):
+    tempdir = tempfile.TemporaryDirectory()
+
+    with tarfile.open(output_filename, 'w') as ctf_tar:
+
+        for component_descriptor in component_descriptors:
+            cd_filename = component_descriptor.component.name.replace("/", "_")
```

Comment:
nit: consistently use single or double-quotes (single-quotes are preferred in cc-utils)